src/tools/gophish_export.py

```diff
@@ -94,25 +116,30 @@ def pull_gophish_groups(api, assessment_id):
 
 
 def campaignControl(api, assessment_id):
-    """Control the campaign importing of an assessment to DB."""
-    # STAND ALONE
+    """Control the campaign importing of an assessment to DB.
+
+    Results are saved to a json file.
+
+    Args:
+        api (GoPhish API): Connection to GoPhish server via the API.
+        assessment_id (string): Assessment identifier to get campaigns from.
+
+    Returns:
+        List of the assessment's campaigns with data.
+    """
     campaignIDs = pull_gophish_campaign(api, assessment_id)
     campaigns = list()
 
     for campaign_id in campaignIDs:
         campaigns.append(import_campaign(api, campaign_id))
 
-    logging.info("Successfully added campaigns for {}".format(assessment_id))
+    logging.info(f"Successfully added campaigns for {assessment_id}")
```

Comment:
See similar comment above about the logging statement in `import_users`.

Reply:
This is showing as f-string in fa6ca50.

Reply:
Changed wording in fefb0b4.

Reply:
The wording here should match the change you made in https://github.com/cisagov/gophish-tools/commit/52228864d25b4500ee4eb7c3321e7c34235b3c18#diff-6bec27233cb25f407c10f9c1ced80f3aR98, so I'd change it to:
```
logging.info(f"{len(campaigns)} campaigns found for assessment {assessment_id}.")
```

Reply:
Changed word in 64f8559.